### PR TITLE
FF-67 basic blog post and cleaning up

### PR DESCRIPTION
**Pull request by qvalentin**
@open-schnick maybe you can add something about our deployment solution to the blog.

### CODE REVIEW

File: _posts/2020-10-24-first-usecases.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+title: "First use cases in our wiki"
+author: valentin
+categories: docu
+issuelink: https://github.com/FileFighter/filefighter.github.io/issues/17
+---
+
+
+Hi there, this blog post will be about the documentation of use cases.
+
+For this week we got the task to create Use-Case Specifications for two of our use cases. We choose the [permissions use case](/wiki/usecases/crudPermissions.html) for sharing files and directories and the [view folder contents](/wiki/usecases/viewFolderContents.html) use case. We created the activity diagrams for both use cases using [Draw.io](https://app.diagrams.net/){:target="_blank"} and started making frontend mockups with [Pencil Project](http://pencil.evolus.vn/){:target="_blank"}.
```
Comment (Gimleux) [inline _posts/2020-10-24-first-usecases.md:12]: great praise for linking directly to the usecases

File: wiki/usecases/crudPermissions.md [hunk 2]
```diff
@@ -1,13 +1,14 @@
...
-title: "Use-Case Specification: Share files and directories"
+title: "Use-Case Specification: Share files and directories (Permissions CRUD)"
 notInNav: true
 ---
 
 # 1. Share files and directories
 
 ## 1.1 Brief Description
-tbd
+The owner of a file (the person who uploaded the file) will be able to permit other users to see and edit his files. This will be managed for folders and their content or individual files. He can permit it to single users or a certain group of users (administrators/internal users/external users).
```
Comment (Gimleux) [inline wiki/usecases/crudPermissions.md:10]: Don't think this was part of this week's tasks but thank you for adding it

**Review by Gimleux — APPROVED** [review]

File: wiki/usecases/crudPermissions.md [hunk 3]
```diff
@@ -43,4 +44,7 @@ tbd
 tbd
 
 # 6. Function Points
-tbd
\ No newline at end of file
+tbd
+
+
+For feedback about this page go to this [issue](https://github.com/FileFighter/filefighter.github.io/issues/17){:target="_blank"}.
```
Comment (open-schnick) [inline wiki/usecases/crudPermissions.md:50]: You know you can add `issuelink: https://github.com/FileFighter/filefighter.github.io/issues/17` to the header of the file to automaticly generate the this sentence. i mean except you changed that. :D

File: wiki/usecases/viewFolderContents.md [hunk 4]
```diff
@@ -42,4 +43,7 @@ tbd
 tbd
 
 # 6. Function Points
-tbd
\ No newline at end of file
+tbd
+
+
+For feedback about this page go to this [issue](https://github.com/FileFighter/filefighter.github.io/issues/17){:target="_blank"}.
```
Comment (open-schnick) [inline wiki/usecases/viewFolderContents.md:49]: same here.

**Review by open-schnick — CHANGES_REQUESTED** [review]

**Pull request merged**